README.md

```diff
@@ -47,6 +54,7 @@ $ make run
 
 This will:
 
+* Xargo builds libcore for our x86_64-unknown-intermezzos-gnu target
```

Comment:
Could you say "Build libcore for our x86_64-unknown-intermezzos-gnu target with Xargo", please?